### DISCUSSION TITLE
Plea for contact

### DISCUSSION THREAD

**Discussion by timurlain** [chrispryer/pdappend#20] [General]
Hello @pryborg

My apologies for writing here, but I did not see any other way to contact you.

I saw your comments with regards to solving a special VRP for 3PL  in another repository. This is my area of interest / development and some of the features you mentioned we can match.

If you are interested let me know ideally we could use email, or some more direct form of communication.

My email is tomas.pajonk@solvertech.cz

**Comment by cnpryer** [comment]
Hi! This project is unrelated. Can we chat in https://github.com/pryborg/vrp/discussions ?

**Reply by timurlain** [reply]
Of course, I was concerned  /did not want to hijack the other project with disccussion on a competing / different solution, as that seems odd to me. I will delete this and the previous message, so it does not bother you in a bit.
